am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/TestDataFactory.java

```diff
@@ -14,7 +14,8 @@ private TestDataFactory() {
 
     public static ExplicitAccessRecord createRecord(String resourceId,
                                                     String accessorId,
-                                                    Set<Permission> explicitPermissions) {
+                                                    Set<Permission> explicitPermissions
```

Comment:
Why is that change needed? Please avoid pointless changes to ease review.

Reply:
👍